scripts/base/init-bare.bro

```diff
@@ -3710,11 +3710,36 @@ export {
 	## external harness and shouldn't output anything to the console.
 	const errors_to_stderr = T &redef;
 }
+
+module Fanout;
```

Comment:
I believe that refactoring wish is long overdue, but for now I would suggest PacketFanout as a proper namespace.